package.json

```diff
@@ -4875,6 +4877,622 @@
             }
           }
         ]
+      },
+      {
+        "type": "monovsdbg",
+        "label": ".NET Core using Mono Runtime",
+        "hiddenWhen": "dotnet.debug.serviceBrokerAvailable",
+        "languages": [
+          "csharp"
+        ],
+        "variables": {
+          "pickProcess": "csharp.listProcess",
+          "pickRemoteProcess": "csharp.listRemoteProcess",
+          "pickRemoteDockerProcess": "csharp.listRemoteDockerProcess"
+        },
+        "configurationAttributes": {
+          "launch": {
```

Comment:
The `launch` and `attach` configurations are generated with the generateOptionsSchema script. You will want to add updating this debugger too.

https://github.com/dotnet/vscode-csharp/blob/9caecf2b984b607f3c9a22b93e50a73dfdf04415/src/tools/generateOptionsSchema.ts#L351-L358

Reply:
NOTE: You also need to update the generator to remove `monoDebuggerOptions` for non-mono configurations.

Reply:
Alternatively, you could also undo the Concord-side change to LanchOptions.json, I will leave a comment in the Concord PR with more information.